tests/framework/PostProcessors/InterfacedPostProcessor/test_historySetSnapshot.xml

```diff
@@ -149,10 +148,6 @@
     </MonteCarlo>
   </Samplers>
 
-  <Databases>
-    <HDF5 name="out_db" readMode="overwrite"/>
-  </Databases>
-
```

Comment:
why database is removed?

Reply:
I removed since it was not part of the testing capability